### PR TITLE
Notification section, missing Edit to change status

### PR DESCRIPTION
**Pull request by xbenyx**
(In progress ) It works but it is missing edit to change status from Active to Inactive and use the config API to get 'ChatBot',
  'Discord Webhook', instead of hardcoded

### CODE REVIEW

**Comment by xbenyx** [comment]
It works as expected. 

image

Just found an error when trying to create a notification for the user Admin. Probably is sth with permissions.

File: src/app/users/all-users/all-users.component.html
Comment (zyronix) [inline src/app/users/all-users/all-users.component.html:1]: Please don't ninja change stuff which is not related to pull requests :). This is non functional, so I am fine with it.

**Review by zyronix — APPROVED** [review]

File: src/app/users/all-users/all-users.component.html
Comment (xbenyx) [reply src/app/users/all-users/all-users.component.html:1]: Sorry for the that. As notifications is kind of hidden in the menu I panicked when i saw that the component was missing. lol
